core/src/main/java/bio/terra/pearl/core/service/workflow/EnrollmentService.java

```diff
@@ -114,6 +134,33 @@ public HubResponse<Enrollee> enroll(ParticipantUser user, PortalParticipantUser
         return hubResponse;
     }
 
+    private Enrollee enrollGovernedUser(String portalShortcode, ParticipantUser proxyUser, StudyEnvironment studyEnv, PortalParticipantUser ppUser, UUID preEnrollResponseId) {
+        // Before this, at time of registration we have registered the proxy as a participant user, but now we need to both register and enroll the child they are enrolling
+        RegistrationService.RegistrationResult registrationResult = registrationService.registerGovernedUser(portalShortcode, proxyUser);
+        log.info("Governed user {} registered", registrationResult.participantUser().getId());
+        ParticipantUser mainUser = registrationResult.participantUser();
+        PortalParticipantUser ppMainUser = registrationResult.portalParticipantUser();
```

Comment:
Is there a consistent naming scheme that we can adopt to more quickly differentiate between the different user types? Between `proxyUser` `ppUser`, `mainUser`, `ppMainUser`, and `governedUser`, it took me a bit of time to understand what was happening here.

This applies generally too, not just within this method